docs/Config.md

```diff
@@ -164,6 +164,9 @@ gui:
   # This can be toggled from within Lazygit with the '~' key, but that will not change the default.
   showFileTree: true
 
+  # If true, show the number of lines changed per file in the Files view
+  showNumberOfLineChangesFilesView: false
```

Comment:
Should we add "in" here, i.e. `showNumberOfLineChangesInFilesView`?

(I was also wondering if we should shorten this to `showNumstatInFilesView`, given that `--numstat` is the git option that is being used. But maybe that's too awkward in camel case.)

Reply:
I changed it to showNumstatInFilesView. I like the shorter Version.